server/plugin/plugin.go

```diff
@@ -75,8 +76,7 @@ type Plugin struct {
 
 	router *mux.Router
 
-	telemetryClient telemetry.Client
-	tracker         telemetry.Tracker
+	tracker telemetry.Tracker
```

Comment:
I removed this because it wasn't used in any other place but looking at the doc I realize that at OnDeactivate should be `p.telemetryClient.Close()`.

Do you have any context why that's on the docs but not in the code?

Reply:
self-reply, need to do a close or we'll leak goroutines https://github.com/rudderlabs/analytics-go/blob/master/analytics.go#L100

Reply:
I think I just forgot to close the Client in `OnDeactivate`.